examples.md

```diff
@@ -0,0 +1,722 @@
+\label{eq:1d-column-momentum}
+\end{equation}
+```
+
+Where:
+- $\Pi(\rho\theta) = C_p \left(\frac{R_d \rho\theta}{p_0}\right)^{\frac{R_m}{C_v}}$ is the Exner function
+- $\Phi(z) = gz$ is the geopotential
+
+This is discretized using the following:
+
+```math
+\begin{equation}
+  \frac{\partial \boldsymbol{w}}{\partial t} \approx B\left(-I^f\left(\frac{\rho\theta}{\rho}\right) \nabla^f_v \Pi(\rho\theta) - \nabla^f_v \Phi(z)\right)
+\label{eq:1d-column-discrete-momentum}
+\end{equation}
+```
+
+Where $B$ applies boundary conditions to enforce $\boldsymbol{w} = 0$ at the domain boundaries.
+
+### Prognostic variables
+
+* $\rho$: _density_ measured in kg/m³, discretized at cell centers.
+* $\rho\theta$: _potential temperature density_ measured in K·kg/m³, discretized at cell centers.
+* $\boldsymbol{w}$: _vertical velocity_ measured in m/s, discretized at cell faces.
+
+### Operators
+
+#### Reconstructions
+
+* $I^f$ is the [center-to-face reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateC2F), called `If` in the example code.
+  - Currently this is implemented as the arithmetic mean.
+
+#### Differentiation operators
+
+* $D^c_v$ is the [face-to-center vertical divergence](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.DivergenceF2C), called `∂` in the example code.
+  - This example uses zero vertical velocity at the top and bottom boundaries.
+* $\nabla^f_v$ is the [center-to-face vertical gradient](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.GradientC2F), called `∂f` in the example code.
+* $B$ is the [boundary operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.SetBoundaryOperator), called `B` in the example code.
+  - This enforces zero vertical velocity at domain boundaries.
+
+### Problem flow and set-up
+
+This test case is set up in a vertical column domain from $z=0$ m to $z=30$ km with 30 vertical elements. The column is initialized with a decaying temperature profile, where:
+
+- The virtual temperature starts at $T_{virt\_surf} = 280$ K at the surface
+- It asymptotically approaches $T_{min\_ref} = 230$ K with height
+- The profile follows a hyperbolic tangent function with height
+- The pressure follows a hydrostatic balance equation
+- Density is calculated from the equation of state using virtual temperature and pressure
+
+The initial vertical velocity is set to zero everywhere. To maintain hydrostatic balance, the discrete form computes iteratively the values of density that ensure the vertical pressure gradient balances gravity.
+
+The simulation is run for 10 days to verify that the hydrostatic balance is maintained over time. Results are plotted showing density ($\rho$), vertical velocity ($\boldsymbol{w}$), and potential temperature density ($\rho\theta$) profiles.
+
+## 2D Cartesian examples
```

Comment:
Hi @Snowdog85123 , thank you for your contribution! Everything below this line was already present in the `docs/src/examples.md` file. Hence, it should not be an addition in your PR. 
Perhaps you had a git accident? You can rebase on `main` and it should get rid of the extra addition and only show your actual changes to the file. Thank you